packages/ag-charts-enterprise/src/features/annotations/scenes/channel.ts

```diff
@@ -52,9 +52,20 @@ export class Channel extends Annotation {
         this.updateBackground(datum, top, bottom);
     }
 
-    override toggleHandles(show: boolean) {
+    override toggleHandles(show: boolean | Partial<Record<ChannelHandle, boolean>>) {
+        if (typeof show === 'boolean') {
+            show = {
+                topLeft: show,
+                topMiddle: show,
+                topRight: show,
+                bottomLeft: show,
+                bottomMiddle: show,
+                bottomRight: show,
+            };
+        }
+
         for (const [handle, node] of Object.entries(this.handles)) {
-            node.visible = show;
+            node.visible = show[handle as ChannelHandle] ?? true;
```

Comment:
Possible to avoid the forced cast here?

Reply:
Not without creating lots of extra noise that doesn't really assert any extra safety.